icicle/include/poseidon/constants/generate_parameters.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+
+# 0.3.7
+import galois
+import numpy as np
+
+import os
+
+# pip install poseidon-hash
+from poseidon import round_constants as rc, round_numbers as rn
+import poseidon
+print(f'poseidon.__file__ = {poseidon.__file__}')
+
+# Modify these
+arities = [3, 5, 9, 12]
+#=======
+# field = "m31"
+# p = 2 ** 31 - 1
+# prime_bit_len = 32
+# field_bytes = 4
+#=======
+# field = "babybear"
```

Comment:
why all those coments?
can't you recive a parameter from argv that select the right one

Reply:
See the previous comment.
Will be changed in the future.